src/scythe-bidder/create-room.tsx

```diff
@@ -2,31 +2,53 @@
 
 import React from "react";
 import { jsx } from "@emotion/core";
-import { Button, Card, notification, Select, Form } from "antd";
+import { Button, Card, Form, notification, Select, Switch } from "antd";
 import client from "./client";
-import { MAX_PLAYERS, MIN_PLAYERS, SCYTHE_BIDDER } from "./constants";
+import { FACTIONS_BASE, FACTIONS_IFA, MATS_BASE, MATS_IFA } from "./constants";
+import { MAX_PLAYERS_BASE, MAX_PLAYERS_IFA, MIN_PLAYERS } from "./constants";
+import { SCYTHE_BASE, SCYTHE_BIDDER, SCYTHE_IFA } from "./constants";
 
 export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
   const [numPlayers, setNumPlayers] = React.useState(2);
+  const [ifa, setIfa] = React.useState<boolean>(false);
+  const maxPlayers = ifa === SCYTHE_BASE ? MAX_PLAYERS_BASE : MAX_PLAYERS_IFA;
 
   const onClick = React.useCallback(async () => {
     const numPlayersNum = Number(numPlayers);
+    const newGameType = Boolean(ifa);
```

Comment:
If we go with what I wrote, this won't be needed - just use `if (isIfaActive)` etc.

Reply:
Using isIfaActive now.